composer.json

```diff
@@ -11,13 +11,13 @@
         ]
     },
     "require": {
-        "laravel/framework": "~5.3"
+        "laravel/framework": "~6.0"
     },
     "require-dev": {
-        "mockery/mockery": "0.9.*",
-        "orchestra/testbench": "~3.1",
-        "phpunit/phpunit": "~4.7",
-        "satooshi/php-coveralls": "dev-master@dev",
+        "mockery/mockery": "1.3.*",
+        "orchestra/testbench": "4.*",
+        "phpunit/phpunit": "~8.0",
+        "php-coveralls/php-coveralls": "2.2.*",
```

Comment:
satooshi/php-coveralls is depreceted so replaced by the suggested replacement which worked out fine